src/renderer/services/deposit/utils.ts

```diff
@@ -3,6 +3,7 @@ import {BLSPubkey as BLSPubKey, DepositData, DepositMessage} from "@chainsafe/lo
 import {createHash} from "crypto";
 import {config} from "@chainsafe/lodestar-config/lib/presets/mainnet";
```

Comment:
Oh, we shouldn't assume mainnet config here

Reply:
I guess it was auto-import. Should use the one from `networks` ?

Reply:
yeah